ui/store/background-connection.ts

```diff
@@ -38,6 +38,7 @@ export function submitRequestToBackground<R>(
 type CallbackMethod<R = unknown> = (error?: unknown, result?: R) => void;
 
 /**
+ * @deprecated
```

Comment:
The first line of a JSDoc comment is typically a one-line summary of what the method does. Perhaps this tag could go below with the other tags?

Also, the old tag had additional context about what the user should use instead:

```
@deprecated Use async `submitRequestToBackground` function instead.
```

Perhaps we can restore that text as well

Reply:
Good catch ! will restore it !